### DISCUSSION TITLE
{{ "main" }} in layouts.html

### DISCUSSION THREAD

**Discussion by mshiner** [thuliteio/doks#139] [Q&A] [answered]
There are two variables used in the main block of layouts.html that don't seem to do anything..

Line 5  <h1 class="mt-0">{{ .Title }}</h1>
Nothing appears unless I add a title variable to config.toml which isn't in the default (unless I'm missing something).

Line 8  <p class="lead">{{ .Params.Lead | safeHTML }}</p>
There doesn't seem to be a lead variable in params.toml either but when I add one nothing appears. Fairly new to Hugo so not sure what the issue is.

Thanks
Martyn

**Comment by h-enk** [comment, marked as answer]
Hey Martyn,

`./layouts/index.html` is the [homepage template](https://gohugo.io/templates/homepage/). 

`./content/_index.md` is where you [add content and front matter to the homepage](https://gohugo.io/templates/homepage/#add-content-and-front-matter-to-the-homepage).

So, when you update the frontmatter (title and lead e.g.) in the latter, it will be parsed by the homepage template and will be reflected in the homepage build.

Hope this clarifies.

**Reply by mshiner** [reply]
Henk

Thanks for that.... was a bit confused but it all makes sense now.

M
